docker/compose/swarm/scale_segmentstore

```diff
@@ -59,6 +59,8 @@ for instance in to_create:
           --label com.docker.stack.namespace='pravega' \
           -p '{port}:{port}' \
           -e HDFS_URL={hdfs_url} \
+          -e TIER2_STORAGE=HDFS \
```

Comment:
Are we hardcoding HDFS as the Tier 2 option we are using for swarm deployments?

Reply:
yes, HDFS is the only option supported tier2 storage for docker swarm deployments.